include/wlr/types/wlr_seat.h

```diff
@@ -251,11 +251,18 @@ struct wlr_seat {
 		// wlr_seat_pointer_request_set_cursor_event
 		struct wl_signal request_set_cursor;
 
-		// wlr_seat_request_set_selection_event
-		struct wl_signal request_set_selection;
+		// Called when an application _wants_ to set the selection (user copies some data).
+		// Compositors should listen to this event and call wlr_seat_set_selection
+		// if they want to accept the client's request.
+		struct wl_signal request_set_selection; // wlr_seat_request_set_selection_event
+		// Called after the requested data source is set for the selection.
```

Comment:
Let's leave "requested" out. The compositor can set the selection to a `wlr_data_source` that hasn't been requested by any client.